dev/integration_tests/ui/lib/main.dart

```diff
@@ -18,7 +18,7 @@ void main() {
               textDirection: TextDirection.ltr,
             ),
             onPressed: () {
-              Navigator.push<Object>(
+              Navigator.push<dynamic>(
                 context,
                 MaterialPageRoute<dynamic>(
```

Comment:
```suggestion
                MaterialPageRoute<Object?>(
```
etc for all `dynamic`s